src/context/TokenContext.ts

```diff
@@ -0,0 +1,22 @@
+import React, { createContext } from "react";
+
+export type UserType = {
```

Comment:
move to src/interfaces/user
if you want to explicitly state this is a ts declaration, use IUser
otherwise just User